pandas/tests/test_base.py

```diff
@@ -1280,13 +1325,18 @@ def test_to_numpy_dtype(as_series):
     obj = pd.DatetimeIndex(['2000', '2001'], tz=tz)
     if as_series:
         obj = pd.Series(obj)
-    result = obj.to_numpy(dtype=object)
+
+    # preserve tz by default
+    result = obj.to_numpy()
```

Comment:
is changing the behavior of DatetimeIndex.to_numpy separable?

Reply:
I'm not sure.